pinot-query-runtime/src/test/resources/queries/UDFAggregates.json

```diff
@@ -150,15 +150,15 @@
       },
       {
         "sql": "SELECT PERCENTILE_TDIGEST(float_col, 50), PERCENTILE_TDIGEST(double_col, 5), PERCENTILE_TDIGEST(int_col, 75), PERCENTILE_TDIGEST(long_col, 75) FROM {tbl}",
-        "outputs": [[1.75, 1.0, 137, 137]]
+        "outputs": [[1.75, 1.0, 137.75, 137.75]]
```

Comment:
Here we are changing the return type, right? Couldn't this produce problems in production? Does it only happen with PERCETILE_TDIGEST or are other aggregations that can change their type?

Reply:
Only `PERCENTILE_TDIGEST` where we registered the wrong return type before. This is actually a bugfix.